modules/docs/arrow-docs/docs/docs/arrow-fx/integrations/kotlinxcoroutines/README.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: docs-core
+title: kotlinx.coroutines
+permalink: /docs/integrations/kotlinxcoroutines/
+---
+
+*Note* that as of Arrow 0.9.0, we have deprecated support for `Deferred` from `kotlinx.coroutines`. Using `Deferred` as a return type is considered a smell by the library owners (see [here](https://github.com/Kotlin/kotlinx.coroutines/blob/master/docs/composing-suspending-functions.md#async-style-functions) and [here](http://youtrack.jetbrains.com/issue/KT-25620)), and we were not able to make it work consistently against some of the Laws. 
+
+But we have not given up support for suspend functions! If you would like to use `suspend fun`, you can do so using the `arrow-fx` and this module.
+
+# Kotlin Coroutines and runtime support
+
+Due to the current nature of Kotlin Coroutines' implementation we need to make a differentiation between the Coroutines feature in the language and the runtime, in order to understand their differences and this module altogether:
```

Comment:
`between the Coroutines feature in the language and the kotlinx.coroutines library`